json/fermentable.json

```diff
@@ -100,7 +100,50 @@
             },
             "inventory": {
               "$ref": "#/definitions/FermentableInventoryType"
-            }
+            },
+            "glassy": {
+              "description" : "Used to indicate the 'crystallized' percentage of starches for crystal malts.",
+              "$ref": "measureable_units.json#/definitions/PercentType"
+            },
+            "plump": {
+              "description" : "The percentage of grain that masses through sieves with gaps of 7/64 and 6/64, desired values of 80% or higher which indicate plump kernels.",
+              "$ref": "measureable_units.json#/definitions/PercentType"
+            },
+            "half": {
+              "$ref": "measureable_units.json#/definitions/PercentType"
+            },
+            "mealy": {
+              "description" : "The opposite of glassy, a mealy kernel is one that is not glassy. Base malt should be at least 90%, single step mashes generally require 95% or higher.",
+              "$ref": "measureable_units.json#/definitions/PercentType"
+            },
+            "thru": {
+              "description" : "The Percentage of grain that makes it through a thin mesh screen, typically 5/64 inch. Values less than 3% are desired.",
+              "$ref": "measureable_units.json#/definitions/PercentType"
+            },
+            "friability": {
+              "description" : "Friability is the measure of a malts ability to crumble during the crush, and is used as an indicator for easy gelatinization of the grain and starches, as well as modification of the malt. Value of 85% of higher indicates a well modified malt and is suitable for single step mashes. Lower values may require a step mash.",
+              "$ref": "measureable_units.json#/definitions/PercentType"
+            },
+            "di_ph": {
+              "description" : "The pH of the resultant wort for 1 lb of grain mashed in 1 gallon of distilled water. Used in many water chemistry / mash pH prediction software.",
+              "$ref": "measureable_units.json#/definitions/AcidityType"
+            },
+            "viscosity": {
+              "description" : "The measure of wort viscosity, typically associated with the breakdown of beta-glucans. The higher the viscosity, the greater the need for a glucan rest and the less suitable for a fly sparge.",
+              "$ref": "measureable_units.json#/definitions/ViscosityType"
+            },
+            "dms_p": {
+              "description" : "The amount of DMS precursors, namely S-methyl methionine (SMM) and dimethyl sulfoxide (DMSO) in the malt which convert to dimethyl sulfide (DMS).",
+              "$ref": "measureable_units.json#/definitions/ConcentrationType"
+            },
+            "fan": {
```

Comment:
@RossMerr 

Should there be an additional property for fermentability that got lost in a copy-paste?

Reply:
Well originally I added 13 properties onto FermentableBase, I moved 11 into FermentableType as it already had kolbach_index and protein so we got left with 11

I guess it was the protein already being their that's causing the confusion.